### PR TITLE
convert_vm_to_ovirt: assign None to parameters for hypervisor kvm

### PR DESCRIPTION
**Pull request by chloerh**
And remove parameters that are no longer needed.

### CODE REVIEW

File: v2v/tests/src/convert_vm_to_ovirt.py [hunk 1]
```diff
@@ -36,7 +36,6 @@ def run(test, params, env):
     vpx_pwd_file = params.get("vpx_pwd_file")
     vpx_dc = params.get("vpx_dc")
     esx_ip = params.get("esx_ip")
-    hypervisor = params.get("hypervisor")
```
Comment (will-Do) [inline v2v/tests/src/convert_vm_to_ovirt.py]: Why remove this var

Comment (chloerh) [inline v2v/tests/src/convert_vm_to_ovirt.py]: It's duplicate with line 26

**Comment by chloerh** [comment]
@will-Do add v2v_tiimeout for import to ovirt

**Comment by will-Do** [comment]
ACK

**Pull request merged**